.github/workflows/ci-tk.yaml

```
@@ -22,7 +22,90 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
   cancel-in-progress: true
 
+env:
+  LLVM_SHA_FILE: llvm-sha.txt
+  LLVM_CACHE_NUMBER: 1  # Increase to reset cache
+  WATER_REF: 1a84aa14ee74d5015ceb41bed438de4389f59b83
+
 jobs:
+  build_llvm_linux:
+    name: Builds Linux LLVM/MLIR
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [linux-mi300-1gpu-ossci-iree-org]
```

This is not an "os" (operating system), it is a runner label
```suggestion
        runs-on: [linux-mi300-1gpu-ossci-iree-org]
```